src/list.tsx

```diff
@@ -106,11 +108,13 @@ export const Content: React.FC<ContentProps> = ({
     const centerY = numTextLines === 1;
 
     return (
-        <Box paddingY={16} className={styles.content}>
+        <Box paddingY={16} className={styles.content} id={labelId}>
```

Comment:
Moving the labelId to the entire row. In this way, detail/right is included inside control's content